tutorials/terraform/sqlserver-data-migration.tf

```diff
@@ -0,0 +1,80 @@
+# Infrastructure for Yandex Cloud Managed Service for SQL Server
+#
+# RU: https://cloud.yandex.ru/docs/managed-sqlserver/tutorials/data-migration
+# EN: https://cloud.yandex.com/en/docs/managed-sqlserver/tutorials/data-migration
+#
+# Set the configuration of the Managed Service for SQL Server cluster:
+locals {
+  sql_server_version = "" # Set the SQL Server version. Мust be the same or higher than the version in the source cluster.
```

Comment:
Set the SQL Server version. It must be the same or higher than the version in the source cluster.